### ISSUE TITLE
Error while requiring module nuxt-highcharts: Error: Cannot find module './shared/untyped.a47b2336.cjs'

### ISSUE THREAD

**Issue by Phanuwit** [unjs/untyped#123]
### Environment

node v18.18.2 (npm v9.8.1)

### Reproduction

https://github.com/richardeschloss/nuxt-highcharts

### Describe the bug

Unable to use yarn command.
<img width="792" alt="Screenshot 2567-01-20 at 00 57 46" src="https://github.com/unjs/untyped/assets/24976372/dc7d724b-ffe8-4398-93cc-3e9b29307c5f">

**Comment by iford** [comment]
I have the same problem.

**Comment by jgourmelen** [comment]
same for me

**Comment by stevenhurth** [comment]
I put in a hard dependency on 1.4.0 until this gets resolved.

**Comment by galaxyblur** [comment]
Getting this with `nuxt prepare` as well

**Comment by galaxyblur** [comment]
```
  "overrides": {
    "untyped": "1.4.0"
  }
```
as a temp fix worked for me

**Comment by angelhdzmultimedia** [comment]
> ```
>   "overrides": {
>     "untyped": "1.4.0"
>   }
> ```
> 
> as a temp fix worked for me

I put this in package.json and error keeps coming after deleting .nuxt, node_modules and pnpm-lock.yaml.😔

**Comment by markthree** [comment]
I think it may be caused by this change → [untyped/commit/1038f03](https://github.com/unjs/untyped/commit/1038f03)

I think I should be able to solve this problem 🥰

**Comment by kasperkronborg** [comment]
I believe that's correct @markthree. I was looking at the same thing. There should never be a reason for node to try and resolve a `.d.cts` file.

**Comment by komgrip** [comment]
`"resolutions": {
		"**/untyped": "1.4.0"
	}`


	
fix worked for me.

**Comment by samasarin** [comment]
we have the same issue.
fixed by adding this to my package.json file:
`"resolutions": {
    "untyped": "=1.4.0"
  },`

**Comment by samasarin** [comment]
@pi0 Can you please review this issue.
Our builds are getting failed for this.

![image](https://github.com/unjs/untyped/assets/129737495/f7aef1d1-5a94-44d2-9f60-e5ab7ef306ce)

**Comment by pi0** [comment]
Sorry im away from my laptop till tonight. Will check to fix asap.

**Comment by Alkiyogoma** [comment]
```
"overrides": {
    "untyped": "1.4.0"
   }
 ``` 

 ### Resolved
  Installed it as dependency instead of override, working now.

**Comment by pi0** [comment]
Released 1.4.2 sorry for inconveniences 🙏

https://github.com/unjs/untyped/releases/tag/v1.4.2

**Comment by jgourmelen** [comment]
thanks !!
